CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Bump `@graphiql/plugin-explorer` version to `^1`
```

Comment:
where's the bump itself?

Reply:
In the HTML itself (`app/graphiql.html`). The version within the CDN URLs was bumped, e.g.:

```diff
- https://unpkg.com/@graphiql/plugin-explorer@0.3/dist/index.umd.js
+ https://unpkg.com/@graphiql/plugin-explorer@1/dist/index.umd.js
```